LandSandBoat/server
# Move conquest system to world server

**_I affirm:_**

## What does this pull request do?

- Weekly tally / hourly updates now happen on world server
- World server communicates of map server updates via zmq
- Adds xi_world configuration and debug symbols for Mac OS
- Maps cache all data. Only world server uses DB.

## Steps to test these changes

Kill stuff as level 15+ outside. Check that influence points are changed in db. Check region map.
Run !updateconquest 0 for tally.
Run !updateconquest 2 for conquest update, after killing lots of mobs.
